wazuh/wazuh-dashboard-plugins
# Set a fixed width to the default columns on each selected field

### Description
Set a fixed width to the default columns on each selected field
 
### Issues Resolved
https://github.com/wazuh/wazuh-dashboard-plugins/issues/7059

### Evidence
<details>
  <summary>AWS</summary>

![image](https://github.com/user-attachments/assets/2eea16aa-701c-4d9c-970f-3094fa369454)

</details>
<details>
  <summary>Docker</summary>

![image](https://github.com/user-attachments/assets/4302c887-0919-47fb-9089-17beaeb06a35)

</details>

<details>
  <summary>FIM</summary>

![image](https://github.com/user-attachments/assets/239fb9fc-09ce-4bfe-a9b9-9870fe05f738)

</details>

<details>
  <summary>GDPR</summary>

![image](https://github.com/user-attachments/assets/b96af15b-fca3-4449-a671-914699c172f0)

</details>

<details>
  <summary>Github</summary>

![image](https://github.com/user-attachments/assets/c28a8ff5-24be-4633-b890-2d7ff973c43a)

</details>

<details>
  <summary>Google Cloud</summary>

![image](https://github.com/user-attachments/assets/e71bc8ce-81bf-482b-a772-b46b4327db52)

</details>

<details>
  <summary>HIPAA</summary>

![image](https://github.com/user-attachments/assets/ef958d14-8fcc-4ef3-af65-0bef202caf5a)

</details>

<details>
  <summary>Malware Detection</summary>

![image](https://github.com/user-attachments/assets/9f23ad78-cc3f-4c8c-844b-2ce91f38bb0d)

</details>

<details>
  <summary>MITRE ATT&CK</summary>

![image](https://github.com/user-attachments/assets/1f9cac25-7455-4d34-b388-a09f2e6622a0)

</details>

<details>
  <summary>NIST</summary>

![image](https://github.com/user-attachments/assets/71330e20-573f-4290-a10f-193c6963c9ba)

</details>

<details>
  <summary>Office 365</summary>

![image](https://github.com/user-attachments/assets/43c6f0e4-e7ac-4c17-bda6-0368f1cf49a7)

</details>

<details>
  <summary>PCI DSS</summary>

![image](https://github.com/user-attachments/assets/23319708-1c81-4b0b-a0f9-ec04be90a6e7)

</details>

<details>
  <summary>Threat Hunting</summary>

![image](https://github.com/user-attachments/assets/f783c007-1c90-488c-8524-c669c0c61378)

</details>

<details>
  <summary>TSC</summary>

![image](https://github.com/user-attachments/assets/77a2e30f-38cf-4b5e-8823-df8c9d7f3a87)

</details>

<details>
  <summary>Vulnerabilities Detection</summary>

![image](https://github.com/user-attachments/assets/a210ebf3-24ac-4bda-a386-127f360d2bdc)

</details>


#### 

### Test
1. Navigate to the Events panel for each Application.
2. Verify that the columns have a fixed width for the small columns and allow the large column to expand.

### Check List
